### PR TITLE
hack/munich22: Detail challenge types

### PR DESCRIPTION
**Pull request by razvand**
Detail challenge types for Munich hackathon (October 22-23, 2022).

### CODE REVIEW

File: content/en/community/hackathons/2022-10-munich/challenges/index.md [hunk 1]
```diff
@@ -21,3 +21,35 @@ We recommend you start small then move up to more difficult challenges.
...
+### Challenge Types
+
+Challenge types generally fall in the categories below.
+You can go beyond the items marked in the [`Hackathons` project](https://github.com/orgs/unikraft/projects/29).
+
+1. **Submitting issues**
+
+   5 hackathon points are awarded for each submitted issue.
+
+1. **Building applications as static PIE binaries** and adding the to the [`static-pie-apps` repository](https://github.com/unikraft/static-pie-apps)
```
Comment (RaduNichita) [inline content/en/community/hackathons/2022-10-munich/challenges/index.md:34]:
```suggestion
1. **Building applications as static PIE binaries** and adding it to the [`static-pie-apps` repository](https://github.com/unikraft/static-pie-apps)
```

File: content/en/community/hackathons/2022-10-munich/challenges/index.md [hunk 2]
```diff
@@ -21,3 +21,35 @@ We recommend you start small then move up to more difficult challenges.
...
+
+   These applications needn't run on Unikraft.
```
Comment (RaduNichita) [inline content/en/community/hackathons/2022-10-munich/challenges/index.md:36]:
```suggestion
   These applications don't have to run on Unikraft.
```

File: content/en/community/hackathons/2022-10-munich/challenges/index.md [hunk 3]
```diff
@@ -21,3 +21,35 @@ We recommend you start small then move up to more difficult challenges.
...
+   If something doesn't work submit an issue and get points for that.
+
+   30-50 hackathon points are awarded for each submitted application, depending on its complexity.
+
+1. **Fixing build warning messages**
+
+   5-10 hackathon points are awarded for each fix.
```
Comment (RaduNichita) [inline content/en/community/hackathons/2022-10-munich/challenges/index.md:43]:
```suggestion
   5-10 hackathon points are awarded for each fix. depending on its complexity.
```

**Review by RaduNichita — CHANGES_REQUESTED** [review]

File: content/en/community/hackathons/2022-10-munich/challenges/index.md [hunk 4]
```diff
@@ -21,3 +21,35 @@ We recommend you start small then move up to more difficult challenges.
 We will keep the scoreboard [here](https://leaderboardhq.com/3wt61mcc).
 
 Please check the community guidelines on [submitting changes](https://github.com/unikraft/docs/blob/main/content/en/docs/contributing/submitting-changes.md) and [the review process](https://github.com/unikraft/docs/blob/main/content/en/docs/contributing/review-process.md).
+
+### Challenge Types
+
+Challenge types generally fall in the categories below.
```
Comment (RaduNichita) [inline content/en/community/hackathons/2022-10-munich/challenges/index.md:27]:
```suggestion
Challenge types generally fall into the categories below.
```

**Review by RaduNichita — CHANGES_REQUESTED** [review]

File: content/en/community/hackathons/2022-10-munich/challenges/index.md (hunk 3 above)
Comment (razvand) [reply content/en/community/hackathons/2022-10-munich/challenges/index.md:43]: It's implied.

**Review by RaduNichita — APPROVED** [review]
All good now. Thanks for the great help, @razvand

Reviewed-by: Radu Nichita [radunichita99@gmail.com](mailto:radunichita99@gmail.com)

File: content/en/community/hackathons/2022-10-munich/challenges/index.md [hunk 5]
```diff
@@ -20,4 +20,36 @@ We recommend you start small then move up to more difficult challenges.
...
+Challenge types generally fall into the categories below.
+You can go beyond the items marked in the [`Hackathons` project](https://github.com/orgs/unikraft/projects/29).
+
+1. **Submitting issues**
+
+   5 hackathon points are awarded for each submitted issue.
+
+1. **Building applications as static PIE binaries** and adding them to the [`static-pie-apps` repository](https://github.com/unikraft/static-pie-apps)
+
+   These applications aren't require to run on Unikraft.
```
Comment (StefanJum) [inline content/en/community/hackathons/2022-10-munich/challenges/index.md:36]:
```suggestion
   These applications aren't required to run on Unikraft.
```

**Review by StefanJum — COMMENTED** [review]
Looks good, only one comment @razvand.
I will merge after that.

**Comment by razvand** [comment]
> Looks good, only one comment @razvand. I will merge after that.

@StefanJum , solved

**Review by StefanJum — APPROVED** [review]
All good, thanks @razvand.

Approved-by: Stefan Jumarea <stefanjumarea02@gmail.com>

**Pull request merged**